packages/core-components/src/components/Avatar/utils.ts

```diff
@@ -27,9 +27,11 @@ export function stringToColor(str: string) {
   return color;
 }
 
-export function extractInitials(value: string) {
-  return value
-    .match(/(?<!\p{L})\p{L}/gu)
-    ?.join('')
-    .slice(0, 2);
+export function extractInitials(name: string) {
+  const names = name.split(' ');
```

Comment:
I think it could be good to throw in a .trim here just in case

Reply:
Added a test for it too